examples/add_picture_claim_from_idp_metadata.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Set picture claim from IdP picture metadata
+ *
+ * Flow: Complement token, Trigger: Pre Userinfo creation, Pre access token creation
+ *
+ * @param ctx
+ * @param api
+ */
+function add_picture_claim_from_idp_metadata(ctx, api) {
+  // return if picture already set
+  if (ctx.v1.claims && ctx.v1.claims.picture) {
+      return;
+  }
+
+  const metadata = ctx.v1.user.getMetadata();
+
+  for (let i in metadata.metadata) {
```

Comment:
The for loop can be shortened to this one-liner
```ts
metadata.metadata.forEach(({ key, value: picture }) => key === 'idpPicture' && picture && api.v1.claims.setClaim('picture', picture));
```
or with if-condition / ternary op
```ts
// if-condition
metadata.metadata.forEach(({ key, value: picture }) => {
  if (key === 'idpPicture' && picture) {
    api.v1.claims.setClaim('picture', picture);
  }
});

// ternary op
metadata.metadata.forEach(({ key, value: picture }) => key === 'idpPicture' && !!picture ? api.v1.claims.setClaim('picture', picture) : null);
```
Feel free to omit the assignment of `value` to `picture` if you don't want to clutter your mem.